src/Configuration/Project.php

```diff
@@ -89,7 +89,7 @@ public function jira(string $namespace, string $project): self
     {
         $this->issues = "https://{$namespace}.atlassian.net/browse/{$project}-%s";
 
-        $this->assignees = "https://{$namespace}.atlassian.net/secure/ViewProfile?name=%s";
+        $this->assignees = "https://{$namespace}.atlassian.net/secure/ViewProfile.jspa?name=%s";
```

Comment:
I'm not sure if we want to have `?name=%s` here, or use `?email=%s` or `?username=` (name doesn't seem to work, at least from my usage).